docs/User/SWCI_APIs.md

```diff
@@ -38,7 +38,7 @@ The python3 program needs to import the SWCI class, and then use the below APIs.
 |`listTasks()`|This method displays the list of tasks that are registered into the Swarm Learning network.| |
```

Comment:
For assignTask() - 

Assigns a task to a taskrunner instance and specify the minimum peer count for declaring the result. This triggers SWOPs to execute this task<br>- `taskName`: Specifies the unique name of task.<br>- `trName`: Specifies the unique name of Taskrunner. SWOP’s listening on this Taskrunner participates and executes this task.<br>- `WITH <peersNeeded> PEERS`:<br>`peersNeeded` is a nonzero positive integer specifying the minimum number of peers required to complete this task.<br>

Meaningful value of minimum peers is dependent on the task type. For RUN\_SWARM task, it is the number of SL and ML node pairs. For all other TASK types, it is the number of SWOP nodes.<br>- For RUN_SWARM task type, the actual numbers of SL/ML peers started could be equal or greater than `peersNeeded`, depending on the number of the SL nodes defined in the SWOP profiles.<br>- For other TASK types, the number of SWOPs participating would be equal to `peersNeeded`